staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -244,6 +244,8 @@ message Mount {
     // - when set to true, readonly must be explicitly set to true, and propagation must be PRIVATE (0).
     // - (readonly == false && recursive_read_only == false) does not make the mount read-only.
     bool recursive_read_only = 8;
+    // OciMount specifies a special mount type for OCI volume mounts.
```

Comment:
```suggestion
    // OCIVolumeMount specifies a special mount type for OCI volume mounts.
```

Reply:
This was referring to the generated `OciMount` variable, but I removed it completely to avoid confusion.